Resources/Public/Css/Backend/ckeditor.css

```diff
@@ -0,0 +1,7 @@
+.cke .cke_combo_text {
+    width: auto;
+}
+
+.cke.cke_combopanel {
+    width: 300px;
```

Comment:
Please have a look at https://github.com/benjaminkott/bootstrap_package/blob/master/Resources/Public/Scss/RTE/rte.scss#L12-L14. This should be changed here.

Reply:
@gilbertsoft No, changing the code here has no effect to the RTE buttons. The styles you include with "contentsCss" are just used for the main RTE window (where the editor changes the text).

Reply:
Mmh, so I guess we are missing something else here too. Perhaps the registering of the final CSS as done below.

Reply:
I am pretty sure you can't change the "cke_combopanel" via the "rte.css". 
Do you need more infos from me to merge this PR?

Reply:
I have to review and test your changes, perhaps there come up more questions then...